index.js

```diff
@@ -14,6 +14,11 @@ function Server (options) {
   options.http = options.http || 80;
   options.https = options.https || null;
   options.changeOrigin = options.changeOrigin || false;
+  if (options.rewriteHost) {
+    options.rewriteHost = options.rewriteHost + ':' + options.http;
```

Comment:
If the proxy is listening on an non-standard port, then it should explicitly inform the client which port to follow onto.